src/diffusers/models/attention.py

```diff
@@ -309,13 +311,14 @@ def forward(
             norm_hidden_states = self.norm1(hidden_states)
         elif self.norm_type == "ada_norm_continuous":
             norm_hidden_states = self.norm1(hidden_states, added_cond_kwargs["pooled_text_emb"])
-        elif self.norm_type == "ada_norm_single":
+        elif self.norm_type in ["ada_norm_single", "layer_norm_latte"]:
             shift_msa, scale_msa, gate_msa, shift_mlp, scale_mlp, gate_mlp = (
                 self.scale_shift_table[None] + timestep.reshape(batch_size, 6, -1)
             ).chunk(6, dim=1)
             norm_hidden_states = self.norm1(hidden_states)
             norm_hidden_states = norm_hidden_states * (1 + scale_msa) + shift_msa
-            norm_hidden_states = norm_hidden_states.squeeze(1)
+            if self.norm_type == "ada_norm_single":
```

Comment:
is it possible to decide whether to squeeze or not based on shape of `norm_hidden_states`?